template.tpl

```diff
@@ -757,16 +779,26 @@ function trackEvent(event, trackingObject) {
   log("Tracking fired", url);
 }
 
+function setUser(ruid, encodeValue) {
+  // add an option to whether base64 the passed value or not
```

Comment:
Wasn't it possible to check if the value were base64 encoded here in the code?
Then we could remove the "encode" checkbox, and just do it automatically

We should also remove these comments